Misc/NEWS.d/next/Library/2020-05-18-17-29-30.bpo-40626.NeZufF.rst

```diff
@@ -0,0 +1 @@
+Add h5 file extension as MIME Type application/x-hdf5, as per HDF Group recommendation for HDF5 formatted data files.
```

Comment:
```suggestion
Add h5 file extension as MIME Type application/x-hdf5, as per HDF Group recommendation for HDF5 formatted data files. Path contributed by Mark SCHWAB.
```

You can also add an entry at https://github.com/python/cpython/blob/master/Misc/ACKS